diff --git a/nixos/modules/security/acme/default.nix b/nixos/modules/security/acme/default.nix
index 286814eab..dbe45878a 100644
--- a/nixos/modules/security/acme/default.nix
+++ b/nixos/modules/security/acme/default.nix
@@ -414,7 +414,7 @@ let
         requires = [ "acme-setup.service" ];
 
         # https://github.com/NixOS/nixpkgs/pull/81371#issuecomment-605526099
-        wantedBy = lib.optionals (!config.boot.isContainer) [ "multi-user.target" ];
+        wantedBy = lib.optionals (!config.boot.isContainer) [ "deferred-init.target" ];
 
         path = with pkgs; [
           lego
