diff --git a/nixos/modules/services/web-servers/nginx/default.nix b/nixos/modules/services/web-servers/nginx/default.nix
index bf6585645..106d2f2c3 100644
--- a/nixos/modules/services/web-servers/nginx/default.nix
+++ b/nixos/modules/services/web-servers/nginx/default.nix
@@ -1479,7 +1479,7 @@ in
 
     systemd.services.nginx = {
       description = "Nginx Web Server";
-      wantedBy = [ "multi-user.target" ];
+      wantedBy = [ "deferred-init.target" ];
       wants = concatLists (map (certName: [ "acme-finished-${certName}.target" ]) vhostCertNames);
       after =
         [ "network.target" ]
@@ -1593,7 +1593,7 @@ in
       in
       mkIf (cfg.enableReload || vhostCertNames != [ ]) {
         wants = optionals cfg.enableReload [ "nginx.service" ];
-        wantedBy = sslServices ++ [ "multi-user.target" ];
+        wantedBy = sslServices ++ [ "deferred-init.target"  ];
         # Before the finished targets, after the renew services.
         # This service might be needed for HTTP-01 challenges, but we only want to confirm
         # certs are updated _after_ config has been reloaded.
